roadmap.md

```diff
@@ -23,26 +23,40 @@ implement the spec.
 1. Create and deploy a website that features a simple overview, email list and directs visitors to Github.
 1. Store all website assets in the CloudEvents repository, under the governance
 of the working group.
-
-*0.2*
-
 1. Have at least 2 implementations of the specification that can demonstrate interoperability.
 1. Include a specification for mapping the CloudEvents specification to HTTP.
 1. Include a specification for mapping the CloudEvents specification to JSON.
 1. Changes to the spec to facilitate adoption.
-1. Draft documentation and developer guide.
 1. Publicize at conferences ([CloudNativeCon Europe](https://events.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2018/)).
-
-*0.3*
-
 1. Interoperability demo.
     1. At least one open source implementation of sending and receiving events.
     1. Events are sent by code written by Developer1 and recieved by code written by Developer2, where Developer1 has no knowledge of Developer2.
     1. Draft trigger specification (e.g. how events + actions are associated).
-1. Draft documentation and user guides.
+
+*0.2*
+
+1. Incorporate learnings and feedback from interop demo to support wider
+   adoption.
+1. Draft documentation, developer and/or user guide.
+1. Resolve all known large design issues (excluding security issues)
+1. Resolve all known "proposed required attributes" issues
+1. Interoperability demo 2
+    1. Details to be determined
+	1. Showcase demo at conferences - perhaps KubeCon NA 2018
+1. Define the set of protocol and serialization mappings we're going to
+   produce for 0.4 milestone
+
+*0.3*
+
+1. Resolve all known "proposed optional attributes" issues
+1. Resolve all known "security related" issues
 
 *0.4*
+1. Complete the proposed set of protocol and serialization mappings
+1. Resolve "Process" related issues
+
+*0.5*
 
-1. Incorporate learnings and feedback from interop demo to support wider adoption.
+1. Resolve outstanding clarifications and non-semantic issues
 1. Collaborate on libraries and supporting tools to use CloudEvents and
```

Comment:
this is pretty vague... could mean a huge number of things!  maybe "define and prioritize libraries and supporting tools that will accelerate adoption of CloudEvents" ?

Reply:
done